examples/integration_test.py

```diff
@@ -313,11 +314,20 @@ def verify_avro():
                   (msg.topic(), msg.partition(), msg.offset(),
                    msg.key(), msg.value(), tstype, timestamp))
 
+            # verify Falsy keys and values are consumed
+            if not msg.key() is not None and not msg.key():
```

Comment:
The double negations makes this hard to read.

Reply:
makes sense, in my mind if further illustrated the fact that you need to check for None not Falsiness